src/Entity/PublicKeyCredentialSource.php

```diff
@@ -71,17 +70,24 @@ public function __construct(
             $userHandle,
             $counter
         );
-        $this->fmt = $fmt;
     }
 
-    public function getFmt(): string
+    /**
+     * Warning: the id field is accessed directly in :\Webauthn\CeremonyStep\CheckAllowedCredentialList::process
+     * The entity tracks a numeric auto increment id value, but the CheckAllowedCredentialList expects the
+     * publicKeyCredentialId.
+     */
+    public function __get(string $name): mixed
```

Comment:
I don't judge you. But I tried to explain myself in the comments. Is this not clear enough?

Reply:
Before when the application would still call getters and setters, the getId method did a similar thing. It would return the pub key cred id. 

Now that the third party code only implements public property access, we needed this nasty trick.

An alternative would be to reorganize our entity, which would lead to a migration on the production data. Which our PM was not interested in at this point.